src/sentry/conf/server.py

```diff
@@ -635,7 +635,6 @@ def create_partitioned_queues(name):
 
 SENTRY_FEATURES = {
     'auth:register': True,
-    'auth:twofactor': True,
```

Comment:
i can't find it right now but i thought we also exposed this to javascript somewhere.